packages/cli-tool/.eslintrc

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": ["@nimblehq/eslint-config-nimble"],
+  "rules": {}
```

Comment:
If we are not overriding any rules, do we need this line? 🤔

Reply:
Removed in [f38eae3](https://github.com/nimblehq/react-templates/pull/95/commits/f38eae3e78dd3694a8127fa0d676b7a054843f9c) 😅